community/php7/php7-fpm.initd

```diff
@@ -6,8 +6,6 @@
 # variable "user" and optionally "group" in conf.d file, or in the $fpm_config
 # file (the former has precedence).
 
-: ${name:="PHP FastCGI Process Manager"}
```

Comment:
The question is why. I added this variable for purpose. This runscript is designed to be symlinked for particular applications. It makes sense for me to allow users to modify the name in log messages.